submissions/Nikita_Lemesiev/css_and_html_practice/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Silkscreen&display=swap" rel="stylesheet">
+    <title>Pop-up</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <div id="logo">
+           Kottans
+        </div>
+        <nav id="nav_bar">
+            <ul>
+                <li tabindex="0">Mail</li>
+                <li tabindex="0">Images</li>
+                <li>
+                    <input tabindex="0"; id="menu" type="checkbox" name="popup_starter">
+                    <label tabindex="-1" for="menu" id="label_menu">
+                        <div id="label_icon"></div>
+                    </label>
+                        <div id="popup_body">
+                            <div tabindex="0" class="item">
+                                <img src="apps.png" alt="Apps">
```

Comment:
You can have an interesting read about `alt` for images [here](https://www.deque.com/blog/great-alt-text-introduction/#:~:text=It%27s%20increasingly%20common%20to%20see,icons%20to%20be%20entirely%20decorative).

In a few words: usually, if you have an icon that is actually named by some text nearby, like here, you don't need an alt for such a decorative image. It doesn't contain "information" or "content" and just serves decorative pupropses.